br-webui/views/network.liquid

```diff
@@ -226,31 +226,19 @@ socket.on("current network config", function(data) {
 // Set companion network configuration according to the option selected by user
 function setCompanionMode() {
 	if (document.getElementById("manual").checked) {
-		if (confirm("Are you sure you want to configure Companion \
-as the default Manual mode? The system will automatically reboot for \
-reconfiguration")) {
+		if (confirm("Are you sure you want to configure the Companion Ethernet interface in Manual mode? The system will automatically reboot for reconfiguration. The static IP address will be 192.168.2.2 after reboot. CLICKING OK MAY RENDER THE SYSTEM UNUSABLE. IF YOU ARE NOT SURE OF WHAT YOU ARE DOING, CLICK CANCEL!")) {
```

Comment:
If you don't like to break lines, you can do:
```
confirm(
      "something"
    + "something"
)
```

Reply:
I think it is an unreasonable burden on the developer to make strings like this. I think the style should apply to code, to avoid long 'oneliners' or call chaining.

Reply:
I've fixed the strings